packages/example/components/ApiForm/hooks/useValidation.ts

```
@@ -0,0 +1,67 @@
+
+export const useValidation = ({ store, validation }: ValidationProps) => {
+  // 获取所有需要验证的字段值
+  const getFieldValues = useCallback(() => {
+    if (!validation) return {};
+
+    return validation.fields.reduce((acc, fieldId) => {
+      const field = store.scope.get(store.fieldsAtom(fieldId));
+      return {
+        ...acc,
+        [fieldId]: {
+          id: fieldId,
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)